rule-audit-client/src/components/capiFeed/CapiResults.tsx

```diff
@@ -11,32 +11,13 @@ import CapiFeedItem from "./CapiFeedItem";
 type IProps = ReturnType<typeof mapStateToProps>;
 
 const articleNumberFormat = new Intl.NumberFormat("en-GB");
+const checkboxId = "capi-results__show-all-articles";
 
-const filterArticles = (
-  articles: (CapiContentWithMatches | undefined)[],
-  showArticlesWithMatchesOnly: boolean
-) =>
-  showArticlesWithMatchesOnly
-    ? (articles.filter(
-        (_) => (_ !== undefined && !_.meta.matches) || _?.meta.matches.length
-      ) as CapiContentWithMatches[])
-    : (articles.filter((_) => _ !== undefined) as CapiContentWithMatches[]);
-
-const CapiResults = ({ articles, isLoading, pagination }: IProps) => {
-  const [
-    showArticlesWithMatchesOnly,
-    setShowArticlesWithMatchesOnly,
-  ] = useState(false);
-  const checkboxId = "checkbox-show-article-matches";
-  const articleArray = Object.values(articles).filter(notEmpty);
-  const filteredArticles = filterArticles(
-    articleArray,
-    showArticlesWithMatchesOnly
-  );
+const CapiResults = ({ articleIds, isLoading, pagination }: IProps) => {
   const totalArticles = pagination
     ? articleNumberFormat.format(
-        filteredArticles.length < pagination.pageSize
-          ? filteredArticles.length
+        articleIds.length < pagination.pageSize
```

Comment:
A slight simplification of this UI to remove some presentation – this anticipates another PR. As this functionality isn't yet used we're safe to do so 👍